docs/jwst/cube_build/main.rst

```diff
@@ -264,28 +270,15 @@ If the  alternative weighting function (set by ``weighting = msm``) is selected
 In this  weighting function the default value for *p* is read in from the cubepar reference file. It can also  be set 
 by the argument ``weight_power=value``.
 
-Other algorithms such as a 3d generalization of the drizzle algorithm
-are also being studied and may provide better performance for some science applications.
-
-..
-	Additional constraints for weighting=MIRIPSF
-	+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
-	For MIRI the weighting function can be adapted to use the  width  of the PSF and LSF in weighting the point cloud members 
-	within the ROIcentered on the spaxel.  The width of the MIRI PSF varies with wavelength, broader for longer wavelengths.
-	The resolving power of  the MRS  varies with wavelength and band.  Adjacent point-cloud elements may in fact originate from
-	different exposures rotated from one another and even from different spectral bands. In order to properly weight the MIRI 
-	data the distances  between the point cloud element and spaxel the distances are determined in the alpha-beta coordinate 
-	system and then normalized by the width of the PSF and the LSF.  To weight in the alpha-beta coordinates system each cube
-	spaxel center must be mapped to the alpha-beta system corresponding to the channel-band of the point cloud member. T
-	he xdistance and ydistances are redefined to mean:
-
-	* xdistance = distance between point in the cloud and spaxel center along the alpha dimension in units of arc seconds
-	* ydistance = distance between point in the cloud and spaxel center along the beta dimension in units of arc seconds
-	* zdistance = distance between point cloud and spaxel center in the lambda dimension in units of microns along the wavelength axis
-
-	The spatial distances are then normalized by PSF width and the spectral distance is normalized by the LSF:
-
-	* xnormalized = xdistance/(width of the PSF in the alpha dimension in units of arc seconds)
-	* ynormalized = ydistance/(width of the PSF in the beta dimension  in units of arc seconds)
-	* znormalized = zdistance/( width of LSF in lambda dimension in units of microns)
+3-D drizzling
+#############
+
+This algorithm for combining data uses a  3d generalization of the classical 2-D drizzle technique. It is used
```

Comment:
let's be consistent in syntax for "3d" vs. "3-d" vs. "3-D" vs. "3D"

Reply:
fixed